index.js

```diff
@@ -728,6 +727,75 @@ module.exports = {
     })
   },
 
+  /**
+   * Gets a statistic on one field at a time
+   * Supports where and geometry filters and group by
+   * @param {String} table to get data from
+   * @param {String} field to generate stats from
```

Comment:
i think its cleaner for now that the cache supports one field at a time at time and the basecontroller can roll them up. From both a code and sql perspective this is way cleaner and easier to write.